### ISSUE TITLE
chgres_cube no longer works on Hera

### ISSUE THREAD

**Issue by GeorgeGayno-NOAA** [NOAA-EMC/UFS_UTILS#210]
Starting with commit 8b25d26, the chgres_cube program is breaking on Hera with this message: _"error while loading shared libraries: libnetcdf.so.15"._

Switching the netcdf and hdf5 libraries as follows seems to fix the problem:

```
-module load hdf5_parallel/1.10.6.release
-module load netcdf_parallel/4.7.4.release
+module load hdf5_parallel/1.10.6
+module load netcdf_parallel/4.7.4
 module load esmf/8.0.0_ParallelNetCDF
```

Not sure why this is.  On all other machines - WCOSS-Cray, WCOSS-Dell, Orion and Jet - it is working fine

**Comment by edwardhartnett** [comment]
@aerorahul this happened when we took the FindNetCDF.cmake out of the submodule and put it into the repo.

Is there something going on with LD_LIBRARY_PATH?

**Comment by aerorahul** [comment]
No idea.

**Comment by aerorahul** [comment]
We don't build shared libraries.
We use the libraries from hpc-stack.
The modules are not using hpc-stack.

**Comment by GeorgeGayno-NOAA** [comment]
Should I just switch to the hpc-stack on Hera?  Is the stack still in beta mode?

**Comment by aerorahul** [comment]
it is not beta, although someone never took the beta out of the name.

**Comment by GeorgeGayno-NOAA** [comment]
The branch at 5cae3ee was compiled on Hera.  All regression tests were run and passed.

**Comment by GeorgeGayno-NOAA** [comment]
The branch at 2f4e877e was compiled on Hera.  All regression tests were run and passed.  And the GDAS initialization utility was successfully run.  Will request a merge.

**Comment by GeorgeGayno-NOAA** [comment]
Merged to 'develop' at bbb662f.  Closing issue.
